advisories/a101963.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Technical Advisory 101963
+advisory: A-101963
+summary: In specific circumstances, a RESTORE operation may restore incorrect data from a backup that contains incorrect metadata related to revision history.
+toc: true
+affected_versions: v22.2.0 to v22.2.8 and v22.1.0 to 22.1.19
+advisory_date: 2023-05-09
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+In the following two circumstances, backups that include [revision history](../stable/take-backups-with-revision-history-and-restore-from-a-point-in-time.html) may record incorrect metadata. The presence of this incorrect metadata could cause a subsequent `RESTORE` operation to write incorrect data:
+
+- A backup of an `UPDATE`-heavy workload captures more than `kv.bulk_sst.max_allowed_overage` of revision history for a given row (64 MB by default). A subsequent `RESTORE` operation could erroneously omit or include a row, or could restore an incorrect version of it.
+- In a cluster running CockroachDB v22.2, the non-public cluster setting `admission.elastic_cpu.enabled` is set to `true`. A subsequent `RESTORE` operation may erroneously omit or include any key with revision history in the backup.
```

Comment:
```suggestion
- In a cluster running CockroachDB v22.2, the non-public [cluster setting](../{{site.versions["stable"]}}/cluster-settings.html) `admission.elastic_cpu.enabled` is set to `true`. A subsequent `RESTORE` operation may erroneously omit or include any key with revision history in the backup.
```

Reply:
I'm not sure it's a good idea to link to the page that shows you how to set a cluster setting when we are talking about a non-public cluster setting you shouldn't be setting. WDYT?

Reply:
That makes sense. However the way this is written leaves open the suggestion that the cluster setting can be changed to `false`. Do we explicitly recommend against this? If so, I think it would be good to say that, along with excluding the docs link.

Reply:
PTAL at the latest version.